examples/numpyarrays/examply.nims

```diff
@@ -0,0 +1,8 @@
+--b:cpp
+--outdir:bin
+--out:examply.so
+--cc:gcc
+--mm:arc
```

Comment:
Why C++ and ARC (and not ORC)?

Reply:
arc is equivalent to orc without cyclic type but both should work.

c++ because I tested calling some openmp loop from python and had some issue with it using C backend and gcc-14. g++ seemed to work fine so I just rolled with it.

Also found out clang and openmp do not like each other (on OSX at least).